src/styles.css

```diff
@@ -22,6 +22,10 @@
   max-height: 80vh;
 }
 
+.formMinWidth {
```

Comment:
Remind me what was this for

Reply:
This ensures that form for all the sections have same width. Please refer https://github.com/IMGIITRoorkee/omniport-frontend-faculty-profile/issues/8 for this.